homeassistant/components/ecobee/strings.json

```diff
@@ -10,7 +10,7 @@
       },
       "authorize": {
         "title": "Authorize app on ecobee.com",
-        "description": "Please authorize this app at https://www.ecobee.com/consumerportal/index.html with PIN code:\n\n{pin}\n\nThen, press Submit."
+        "description": "Please authorize this app at https://www.ecobee.com/consumerportal/index.html and navigating to My Apps then autorize the app with the PIN code:\n\n{pin}\n\nThen, press Submit."
```

Comment:
The verb grammar isn't correct in the sentence now and we're repeating "authorizing".

Why do we need to add more details? If Ecobee changes the layout on the page we need to update again. Isn't the existing text enough?

Reply:
Apologies for the sloppy grammar.  Will update PR after the below.  

The existing text makes it sound like all you have to do is navigate to the provided hyperlink and you will be prompted to enter the PIN.  This leads to users(specially myself) having to hunt down in the Ecobee portal and spend time searching Ecobee documentation (which was not helpful), then searching the internet.  Took ~30 mins to try to add this integration.   

I agree the dependency on the Ecobee site layout isn't ideal.  Perhaps adding a link to the[ Home Assistant ecobee configuration documentation](https://www.home-assistant.io/integrations/ecobee/#configuration) rather than the additional text?

Reply:
All config flows already have a link to the integration docs under the help icon.